SU-SWS/stanford_basic
# D8CORE-2963: updating Decanter version

# READY FOR REVIEW

# Summary
- Updating the Decanter version

# Review By (Date)
- 12/3

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Verify the site appears the same. There shouldn't be any noticeable changes.

# Affected Projects or Products
- SOE and Stanford Basic

# Associated Issues and/or People
- D8CORE-2963 and D8CORE-2964
- https://github.com/SU-SOE/soe_basic/pull/49


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
